blog-meeting-minutes/2024-03-28-security-hour.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: security-office-hour-2024-03-28
+title: Security Office Hour 28.03.2024
+authors: 
+    - rohan_krishnamurthy
+tags: [meeting-minutes, community, security]
+---
+
+## Security Office Hour meeting minutes
+
+### Announcements
+
+- SAST: 
+  -	Veracode - Offboarding: Last reminder, license terminates on 30-March-2024
+  -	CodeQL - Onboarding- Workflow Setup: [TRG 8.01](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-8-01)
+-	DAST security scans are not part of the next release 24.05 (Updates will follow through the QG Acceptance Criteria)
+-	KICS, Trivy, GitGuardian and Dependabot tools will continue as it is.
+
+### Open Discussions
+- Cofinity-X Security contact for incident reporting - Security@cofinity-x.com
```

Comment:
Good to know, but I'm not sure if this might be interpreted as "report issues there" when mentioned here. 
Tractus-X has it's own security findings process, and that's the one we are concerned about

Reply:
Yes @SebastianBezold. This was just the information that the cofinity-X member wanted to share with us. (security team)
 
I hope there won't be misinterpretation. Do you want me to rephrase it for clarity?

Reply:
Done ✔️